SkyhighSecurity/skyhigh_secure_web_gateway/ingest/parser.yml

```diff
@@ -101,14 +101,15 @@ stages:
           '2': 'denied'
           '3': 'error'
         mapping:
-          event.code: event.action
-        fallback: 'denied'
+          event.code: event.type
```

Comment:
Be careful. `event.type` is an array with pre-defined value: https://www.elastic.co/guide/en/ecs/8.2/ecs-allowed-values-event-type.html

Reply:
Good catch ! Thank you !